### DISCUSSION TITLE
Adapter to QTO tool

### DISCUSSION THREAD

**Discussion by fnarch** [BHoM/BHoM#1527] [Q&A]
Hello,
I wondering if is it possible to create a BHoM workflow with adapter components to a QTO tool (transfer data quantities informations). Has anyone done something like this?
Thank you

**Comment by FraserGreenroyd** [comment]
Hi @fnarch , I don't think anyone in BHoM land has done this. @alelom or @al-fisher are either of you aware of anything? If there's a particular tool you're interesting in building an adapter for, I'm sure @alelom and others in the group would be happy to help set something up? 😄

**Reply by fnarch** [reply]
Thank you, I wondering if that's possibile with the excel adapter...
Do you have some example to connect rhino to excel via adapter?
Thank you again

**Reply by alelom** [reply]
Hi @fnarch, I do not know of anything done in this area. For docs on Excel adapter, have a look at the docs (https://bhom.xyz/documentation/) and maybe others like @pawelbaran may have more material on it?
